calendar.js

```diff
@@ -45,11 +71,32 @@ function createTable() {
         tr.id = i;
         table.appendChild(tr);
         var td = document.createElement('td');
+        td.id="hour"+i;
         td.innerText = i + ":00";
         tr.appendChild(td);
 
         for (var j = 1; j < 8; j++) {
             var td = document.createElement('td');
```

Comment:
this variable will have function scope because of hoisting, please declare `td` at the top of the function